### PR TITLE
forms 7.3 | new forms features for 7.3 | LRDOCS-8176

### PR DESCRIPTION
**Pull request by rbohl**
incorporated new information on pages and success pages
also incorporated new information on editing forms entries
https://issues.liferay.com/browse/LRDOCS-8176

### CODE REVIEW

**Comment by rbohl** [comment]
> Files Changed 1,675
If you update your master i can resend ;)

**Comment by rbohl** [comment]
Bump @natocesarrego

File: docs/dxp/7.x/en/process-automation/forms/user-guide/managing-form-entries.md [hunk 1]
```diff
@@ -20,6 +20,31 @@ Once users begin submitting form entries, you can access the form entry data thr
...
+1. Click the _Actions_ (![Actions](../../../images/icon-actions.png)) button for the form and select _Edit_.
+1. Once you finish editing, click _Submit_.
+
+![Updated Form Entries have an incrementally higher version.](./managing-form-entries/images/10.png)
+
+Users need the following permissions to edit Form Entries: 
+
+- Update permission on the Form Instance
+- View permission on the Site and Asset Library Menu
+- Access permission on the Forms application in Site and Asset Library Administration
```
Comment (natocesarrego) [inline docs/dxp/7.x/en/process-automation/forms/user-guide/managing-form-entries.md:40]: @rbohl, if the user desire to update the permissions of a role associated to a single form, then he/she can do it through `Actions > Permissions`. Perhaps, it's worth share it with our audience, what do you think?

**Comment by rbohl** [comment]
Added a commit for the single form instance permission text, forwarded: https://github.com/sez11a/liferay-learn/pull/176

Thanks Renato!

**Pull request closed without merging**